### ISSUE TITLE
How do I force it to work in production

### ISSUE THREAD

**Issue by abstractpoint** [JackAdams/meteor-editable-json#10]
Great package, very useful for creating a quick CRUD interface of sorts.
I have a password protected app and I want to run it when I use --production switch. At the moment it I get error in console: Error: No such template: editableJSON

Thanks.

**Comment by JackAdams** [comment]
Yes.  At the moment, the package is `debugOnly=true` (in the `package.js` file).  That means its code is not even included in a production build of the project.  The package's main use is in [Constellation](http://constellation-demo.meteor.com) (dev console) and, for that, it needs to be `debugOnly`, as it opens up some pretty epic security holes. I've been trying to think of a good way to secure the package and also allow it to be included in production builds (ideally, I'd like to use parts of Constellation to create a zero-work-required, drop-in CRUD interface package), but I haven't implemented that yet.

Thanks for opening the issue.  It's something that's been on my mind for a while.  Unfortunately, I can't promise the necessary refactor will be happening soon (I'm coming into a really busy period), but I do have the outline of a plan (in my head) for how to make this happen.

**Comment by abstractpoint** [comment]
Awesome, Thank you for pointing out `debugOnly` setting, didn't notice that. In terms of security if I just use flow router and template level auth on all templates where `> editableJSON` is included should that be safe enough do you think for a small production app? Or do I still need to wait for refactor? Constellation looks amazing!

**Comment by JackAdams** [comment]
Yep, Constellation is very helpful when developing a project. :-)

You could fork the `babrahams:editable-json` package and remove the `debugOnly=true` flag, so that it gets included in the production build, but this would open up gaping security holes in your app.  **I really don't recommend doing this** until after the refactor (in which I'll include an API for securing your production app).

In the interim, take a look at `houston:admin` or `yogiben:admin` for instant CRUD interface solutions.  These are secure and (I believe) production-ready.
